cli/go-git/go.mod

```diff
@@ -1,32 +0,0 @@
-module github.com/go-git/go-git/cli/go-git
-
```

Comment:
Let's also remove the reference from the dependabot config:
https://github.com/go-git/go-git/blob/master/.github/dependabot.yaml#L17-L22